facia-tool/app/util/Acl.scala

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import com.gu.editorial.permissions.client.{PermissionDenied, PermissionGranted, Permission, PermissionsUser}
+import permissions.Permissions
+import play.api.Logger
+import play.api.libs.json.{JsBoolean, JsValue, Json, Writes}
+import switchboard.SwitchManager
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object Authorization {
+  implicit val authorizationWrites = new Writes[Authorization] {
+    def writes(access: Authorization): JsValue = access match {
+      case AccessGranted => JsBoolean(true)
+      case AccessDenied => JsBoolean(false)}}}
+
+sealed trait Authorization
+object AccessGranted extends Authorization
+object AccessDenied extends Authorization
+
+object AclJson {
+  implicit val jsonWrites = Json.writes[AclJson]
+}
+
+case class AclJson (
+  fronts: Map[String, Authorization],
+  permissions: Map[String, Authorization]
+)
+
+object Acl {
+  def testUser(permission: Permission, switch: String)
+              (email: String): Future[Authorization] = {
+    implicit def permissionsUser: PermissionsUser = PermissionsUser(email)
```

Comment:
`val` maybe? Why does it need to be `implicit`, is the `Permissions.get` using it?